src/data/checkfiles/default_checkfiles/unused-metadata.js

```diff
@@ -2,10 +2,13 @@ import React from "react";
 import axios from "axios";
 import { Chip } from "@material-ui/core";
 
+const maxUnusedMetadataBeforeWarningForNoSpecificQuery = 100000;
```

Comment:
i don't think it's 'unused'... i think it's just... total AVUs in the system?

so more like...
`AVUThresholdBeforeWarningForNoSpecificQuery`
or
`MaximumMetadataBeforeWarningForNoSpecificQuery`
or
`MaximumAVUsBeforeWarningForNoSpecificQuery`
or
`MaxAVUsBeforeWarningForNoSpecificQuery`
or
`MaxMetadataBeforeWarningForNoSpecificQuery`
or
`MaxTotalMetadataBeforeWarningForNoSpecificQuery`

Reply:
good point, changed to `AVUThresholdBeforeWarningForNoSpecificQuery`